src/main/docs/guide/csrf/csrfApis.adoc

```diff
@@ -0,0 +1,9 @@
+The main APIs for CSRF protection are:
+
+* api:security.csrf.CsrConfiguration[]
+* api:security.csrf.filter.CsrFilter[]
+* api:security.csrf.filter.CsrFilterConfiguration[]
+* api:security.csrf.resolvers.CsrfTokenResolver[]
+* api:security.csrf.generator.CsrfTokenGenerator[]
+* api:security.csrf.CsrTokenValidator[]
```

Comment:
Inconsistent naming? Also I think the package name is wrong

Reply:
resolved via https://github.com/micronaut-projects/micronaut-security/pull/1826/commits/1f92c7c3c48866d517e91963e67cec6a054ae8f9